phil/s013-course-periods.md

```diff
@@ -81,6 +81,19 @@ in the export for BE
 ]
 ```
 
+# Learning Guide
```

Comment:
If we were going to use different endpoints, I might suggest `/courses/23/student_guide(/role/:role_id)` and `/courses/23/teacher_guide` -- the former is role-aware but the latter is not.  Alternatively the former could be `students/467/guide` but that would be the only `students` endpoint we have and separates it maybe too much from the teacher guide, which is in the end not all that different.